doozerlib/dblib.py

```diff
@@ -18,47 +70,330 @@ def __init__(self, runtime, environment, dry_run=False):
+
+        """
+        :param table_name
+        This method creates table, if it doesn't exist.
+        """
+
+        table_exist_status = self.check_table_exist(table_name)
+
+        if not table_exist_status:
+            self.runtime.logger.info("Table [{}] not found in database [{}]."
+                                     .format(table_name, self.db))
+            cursor = self.connection.cursor()
+            cursor.execute(f'create table {table_name}(log_{table_name}_id bigint auto_increment primary key)')
+            cursor.close()
+            self._table_column_cache[table_name] = dict()
+            self._table_column_cache[table_name]["log_" + table_name + "_id"] = True
+            self.runtime.logger.info("Successfully create table [{}] in database [{}]."
+                                     .format(table_name, self.db))
+
+    def handle_missing_columns(self, payload, table_name):
+
+        """
+        :param payload
+        :param table_name
+
+        This method creates missing columns, which are present in payload but not in table.
+        """
+
+        missing_columns = self.get_missing_columns(payload, table_name)
+
+        if missing_columns:
+            cursor = self.connection.cursor()
+
+            for column in missing_columns:
+                missing_column = column[0]
+                column_type = column[1][0]
+                cursor.execute(f"alter table {table_name} add column `{missing_column}` {column_type}")
+                self._table_column_cache[table_name][missing_column] = True
+
+                self.runtime.logger.info("Added new column [{}] of identified type [{}] to table [{}] "
+                                         "of database [{}].".format(missing_column, column_type,
+                                                                    table_name, self.db))
+
+            cursor.close()
+
+    def insert_payload(self, payload, table_name, dry_run=False):
+
+        """
+        :param payload
+        :param table_name
+        :param dry_run
+
+        This  method creates table entry for the given payload. If dry_run is true, the insert string is
+        just printed on output.
+
+        The method returns 0, on successful creation of record entry in the table else returns 1.
+        """
+
+        try:
+            columns, values = [], []
+
+            for column in payload:
+                columns.append(f'`{column}`')
+                values.append(f'"{payload[column]}"')
+
+            columns = ",".join(columns)
+            values = ",".join(values)
+
+            insert_string = f"insert into {table_name}({columns}) values({values})"
+
+            if not dry_run:
+                cursor = self.connection.cursor()
+                cursor.execute(insert_string)
+                self.connection.commit()
+                cursor.close()
+                return 0
+            else:
+                print(insert_string)
+                return 0
```

Comment:
minor suggestion - we won't need the else part, plus keeping the shorter return branch in if 

```suggestion
            if dry_run:
                print(insert_string)
                return 0
            
            cursor = self.connection.cursor()
            cursor.execute(insert_string)
            self.connection.commit()
            cursor.close()
            return 0
```